README.md

```diff
@@ -0,0 +1,78 @@
+# kmesh-coredns-plugin
+
+The plugin runs as a standlone server in the cluster, serving DNS A records over gRPC to CoreDNS.
+
+# Quick start
+
+## Step 1: Install kubectl & kind
+
+Linux:
+```sh
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.18.0/kind-linux-amd64
+chmod +x ./kubectl ./kind
+sudo mv ./kubectl ./kind /usr/local/bin
+```
+
+## Step 2: Create and Activate kind
+```sh
+kind create cluster --name kmesh-dns
+```
+
+## Step3: Install Istio
+please refer to [Install istio](https://istio.io/latest/docs/setup/getting-started/#install)
+
+## Step4: Deploy Kmesh coredns plugin
+```sh
+kubectl apply -f manifest/deploy.yaml
+```
+
+## Step5: Forward coreDNS to Kmesh coredns plugin
+```sh
+kubectl edit cm coredns -n kube-system
+```
+
+```yaml
+apiVersion: v1
```

Comment:
Instead of editing this, is there a way to create an additional cm?

Reply:
coreDNS seems only read config from this configmap, we can create a new one, but we have to use k8s Kustomize merge two configmaps together, but it's getting more complicated